docs/gateway-api/gateway-api-v1.md

```diff
@@ -175,6 +175,7 @@ AKO Gateway Implementation follows following naming convention:
   3. Pool                  `ako-gw-<cluster-name>--<sha1 hash of <gateway-namespace>-<gateway-name>-<route-namespace>-<route-name>-<stringified FNV1a_32 hash of bytes(jsonified match)>-<backendRefs_namespace>-<backendRefs_name>-<backendRefs_port>>` 
   4. PoolGroup             `ako-gw-<cluster-name>–-<sha1 hash of <gateway-namespace>-<gateway-name>-<route-namespace>-<route-name>-<stringified FNV1a_32 hash of bytes(jsonified match)>>` 
   5. SSLKeyAndCertificate  `ako-gw-<cluster-name>--<sha1 hash of ako-gw-<cluster-name>--<hostname>`
```

Comment:
naming convention of sslkeyandcertificate will undergo change.

Reply:
Will take that up as part of PR which will change naming convention of secret